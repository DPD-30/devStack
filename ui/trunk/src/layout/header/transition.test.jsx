import React from 'react'
import { renderToString } from 'react-dom/server'


import LayoutHeaderTransition from './transition.jsx'

jest.unmock('./transition.jsx')


test('LayoutHeaderTransition matches snapshot', () => {
	const comp = renderToString(<LayoutHeaderTransition />)

	expect(comp).toMatchSnapshot()
})
