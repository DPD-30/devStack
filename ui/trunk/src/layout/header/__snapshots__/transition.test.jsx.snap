// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LayoutHeaderTransition matches snapshot 1`] = `
<div style="position:absolute;z-index:99;width:100%"
     data-reactroot
>
  <div class="animated fadeInDown"
       style="animation-delay:0ms;animation-duration:850ms;pointer-events:all"
  >
  </div>
</div>
`;
